subprojects/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/single_versions.adoc

```diff
@@ -41,6 +41,22 @@ Versions are ordered based on the following rules:
 ** The string `SNAPSHOT` has **no special meaning**, and is sorted alphabetically like any other string part: `1.0-alpha` < `1.0-SNAPSHOT` < `1.0-zeta` < `1.0-rc` < `1.0`.
 ** Numeric snapshot versions have **no special meaning**, and are sorted like any other numeric part: `1.0` < `1.0-20150201.121010-123` < `1.1`.
 
+[NOTE]
+====
+Gradle 6.5 introduces a number of changes around version sorting, by special casing more suffixes:
```

Comment:
:o: It reads like it's enabled by default. You have to read the full paragraph to realize it's opt-in. Suggestion:

> Gradle 6.5 supports an alternate, opt-in, version sorting scheme which special cases more suffixes